commands/secret_create.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) OpenFaaS Author(s) 2019. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/openfaas/faas-cli/proxy"
+	"github.com/spf13/cobra"
+)
+
+var (
+	secretName  string
```

Comment:
Imho this is better declared within the `runSecretCreate` function (the only place where it is used).

Reply:
function or file?

Reply:
I agree, each of these variables should be local variables of the `runSecretCreate` method, similar to the `gatewayAddress` variable.

Reply:
We haven't done this anywhere else, so I'm not considering this a blocker. Perhaps a nice refactoring exercise for someone?

Reply:
ok, sounds good, i will create a ticket